### ISSUE TITLE
Mark classes that use DisposableScope with UseExperimental

### ISSUE THREAD

**Issue by CherryPerry** [badoo/Reaktive#379]
```
w: /Users/user/IdeaProjects/Reaktive/sample-mpp-module/src/commonMain/kotlin/com/badoo/reaktive/samplemppmodule/store/KittenStoreImpl.kt: (16, 37): This declaration is experimental and its usage should be marked with '@com.badoo.reaktive.annotations.ExperimentalReaktiveApi' or '@UseExperimental(com.badoo.reaktive.annotations.ExperimentalReaktiveApi::class)'
```

**Comment by mayokunthefirst** [comment]
Hello @CherryPerry would love to get on this.

**Comment by CherryPerry** [comment]
Sure, feel free.

**Comment by CherryPerry** [comment]
Not all of them were fixed in #386

```
w: /Users/runner/runners/2.163.1/work/Reaktive/Reaktive/sample-mpp-module/src/commonMain/kotlin/com/badoo/reaktive/samplemppmodule/binder/KittenBinder.kt: (13, 33): This declaration is experimental and its usage should be marked with '@com.badoo.reaktive.annotations.ExperimentalReaktiveApi' or '@UseExperimental(com.badoo.reaktive.annotations.ExperimentalReaktiveApi::class)'
w: /Users/runner/runners/2.163.1/work/Reaktive/Reaktive/sample-mpp-module/src/commonMain/kotlin/com/badoo/reaktive/samplemppmodule/binder/KittenBinder.kt: (22, 26): This declaration is experimental and its usage should be marked with '@com.badoo.reaktive.annotations.ExperimentalReaktiveApi' or '@UseExperimental(com.badoo.reaktive.annotations.ExperimentalReaktiveApi::class)'
> Task :sample-mpp-module:compileKotlinMacosX64
w: /Users/runner/runners/2.163.1/work/Reaktive/Reaktive/sample-mpp-module/src/commonMain/kotlin/com/badoo/reaktive/samplemppmodule/binder/KittenBinder.kt: (25, 17): This declaration is experimental and its usage should be marked with '@com.badoo.reaktive.annotations.ExperimentalReaktiveApi' or '@UseExperimental(com.badoo.reaktive.annotations.ExperimentalReaktiveApi::class)'
w: /Users/runner/runners/2.163.1/work/Reaktive/Reaktive/sample-mpp-module/src/commonMain/kotlin/com/badoo/reaktive/samplemppmodule/binder/KittenBinder.kt: (29, 14): This declaration is experimental and its usage should be marked with '@com.badoo.reaktive.annotations.ExperimentalReaktiveApi' or '@UseExperimental(com.badoo.reaktive.annotations.ExperimentalReaktiveApi::class)'
w: /Users/runner/runners/2.163.1/work/Reaktive/Reaktive/sample-mpp-module/src/commonMain/kotlin/com/badoo/reaktive/samplemppmodule/binder/KittenBinder.kt: (34, 14): This declaration is experimental and its usage should be marked with '@com.badoo.reaktive.annotations.ExperimentalReaktiveApi' or '@UseExperimental(com.badoo.reaktive.annotations.ExperimentalReaktiveApi::class)'
w: /Users/runner/runners/2.163.1/work/Reaktive/Reaktive/sample-mpp-module/src/commonMain/kotlin/com/badoo/reaktive/samplemppmodule/binder/KittenBinder.kt: (38, 25): This declaration is experimental and its usage should be marked with '@com.badoo.reaktive.annotations.ExperimentalReaktiveApi' or '@UseExperimental(com.badoo.reaktive.annotations.ExperimentalReaktiveApi::class)'
w: /Users/runner/runners/2.163.1/work/Reaktive/Reaktive/sample-mpp-module/src/commonMain/kotlin/com/badoo/reaktive/samplemppmodule/store/KittenStoreImpl.kt: (15, 2): This class can only be used with the compiler argument '-Xuse-experimental=kotlin.Experimental'
```

**Comment by mayokunthefirst** [comment]
> Not all of them were fixed in #386
> 
> ```
> w: /Users/runner/runners/2.163.1/work/Reaktive/Reaktive/sample-mpp-module/src/commonMain/kotlin/com/badoo/reaktive/samplemppmodule/binder/KittenBinder.kt: (13, 33): This declaration is experimental and its usage should be marked with '@com.badoo.reaktive.annotations.ExperimentalReaktiveApi' or '@UseExperimental(com.badoo.reaktive.annotations.ExperimentalReaktiveApi::class)'
> w: /Users/runner/runners/2.163.1/work/Reaktive/Reaktive/sample-mpp-module/src/commonMain/kotlin/com/badoo/reaktive/samplemppmodule/binder/KittenBinder.kt: (22, 26): This declaration is experimental and its usage should be marked with '@com.badoo.reaktive.annotations.ExperimentalReaktiveApi' or '@UseExperimental(com.badoo.reaktive.annotations.ExperimentalReaktiveApi::class)'
> > Task :sample-mpp-module:compileKotlinMacosX64
> w: /Users/runner/runners/2.163.1/work/Reaktive/Reaktive/sample-mpp-module/src/commonMain/kotlin/com/badoo/reaktive/samplemppmodule/binder/KittenBinder.kt: (25, 17): This declaration is experimental and its usage should be marked with '@com.badoo.reaktive.annotations.ExperimentalReaktiveApi' or '@UseExperimental(com.badoo.reaktive.annotations.ExperimentalReaktiveApi::class)'
> w: /Users/runner/runners/2.163.1/work/Reaktive/Reaktive/sample-mpp-module/src/commonMain/kotlin/com/badoo/reaktive/samplemppmodule/binder/KittenBinder.kt: (29, 14): This declaration is experimental and its usage should be marked with '@com.badoo.reaktive.annotations.ExperimentalReaktiveApi' or '@UseExperimental(com.badoo.reaktive.annotations.ExperimentalReaktiveApi::class)'
> w: /Users/runner/runners/2.163.1/work/Reaktive/Reaktive/sample-mpp-module/src/commonMain/kotlin/com/badoo/reaktive/samplemppmodule/binder/KittenBinder.kt: (34, 14): This declaration is experimental and its usage should be marked with '@com.badoo.reaktive.annotations.ExperimentalReaktiveApi' or '@UseExperimental(com.badoo.reaktive.annotations.ExperimentalReaktiveApi::class)'
> w: /Users/runner/runners/2.163.1/work/Reaktive/Reaktive/sample-mpp-module/src/commonMain/kotlin/com/badoo/reaktive/samplemppmodule/binder/KittenBinder.kt: (38, 25): This declaration is experimental and its usage should be marked with '@com.badoo.reaktive.annotations.ExperimentalReaktiveApi' or '@UseExperimental(com.badoo.reaktive.annotations.ExperimentalReaktiveApi::class)'
> w: /Users/runner/runners/2.163.1/work/Reaktive/Reaktive/sample-mpp-module/src/commonMain/kotlin/com/badoo/reaktive/samplemppmodule/store/KittenStoreImpl.kt: (15, 2): This class can only be used with the compiler argument '-Xuse-experimental=kotlin.Experimental'
> ```

Just fixed this in the new PR.
